src/renderers/webxr/WebXRManager.js

```diff
@@ -133,7 +133,11 @@ class WebXRManager extends EventDispatcher {
 
 			inputSourcesMap.forEach( function ( controller, inputSource ) {
 
-				controller.disconnect( inputSource );
+				if ( controller ) {
```

Comment:
Is this the check you want to do?

```js
if ( controller !== null ) {
```

Reply:
I think it's: 
```js
if ( controller !== undefined ) {
```
If we want to make the check more explicit like above, we should also update `onSessionEvent()`.

Reply:
Ah, I was confused because "This commit adds a null check prior to disconnecting".

Reply:
Yes, we should definitely be as explicit as possible 👍

Reply:
@Mugen87 Feel free to merge and do the `onSessionEvent()` clean up too 🙏

Reply:
Ah apologies, null/undefined feel the same to me in many cases, but I get there's a difference. Thanks for updating the title and merging this through!